components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -298,14 +301,14 @@ protected void setUserClaim(String claimName, String claimValue, UserStoreManage
 
     }
 
-    protected void triggerNotification(User user, String type, String code, Property[] props) throws
-            IdentityRecoveryException {
+    protected void triggerNotification(User user, String type, String code, Property[] props,
```

Comment:
Since this is a protected method changing the signature might cause extended classes written by devs to break.
Shall we add the new method as an overloaded method?

Reply:
Sure. I will add.

Reply:
Fixed in https://github.com/wso2-extensions/identity-governance/pull/452/commits/a8ba91efcec6576251a12cabed82a4cc2b667695